internal/app/main.go

```diff
@@ -18,6 +18,7 @@ var (
 	settings   *config
 	curContext string
 	log        = &Logger{}
+	plugins    map[string]bool
```

Comment:
not a bit fan of global variables, I've been trying to reduce the usage of global vars in this code...

Reply:
It doesn't look like it belongs to any of them, but seeing how state.StorageBackend is used, I suppose it could be in the state. Otherwise it could use it's own package with a global var, but that seems overkill for a small use case

Reply:
I think we can re-engineer this so we don't need to call this method for the same plugin multiple times.